osu.Game.Rulesets.Taiko/UI/TaikoPlayfieldAdjustmentContainer.cs

```diff
@@ -12,12 +12,19 @@ namespace osu.Game.Rulesets.Taiko.UI
 {
     public partial class TaikoPlayfieldAdjustmentContainer : PlayfieldAdjustmentContainer
     {
-        public const float MAXIMUM_ASPECT = 16f / 9f;
-        public const float MINIMUM_ASPECT = 5f / 4f;
-
         private const float stable_gamefield_height = 480f;
 
-        public readonly IBindable<bool> LockPlayfieldAspectRange = new BindableBool(true);
```

Comment:
This was previously set to false in classic to disable aspect range limitations. With these changes playfield aspect range is always limited to some range and never fully unlocked

Reply:
I believe players wanted the unlock in the first place, which is why I'm not sure where this is coming from.

Reply:
With stable, time range is never fully unlocked:
 - With NM, it's limited to a little over 2:1 by fading notes in beyond that
 - With HD, it's trimmed to 4:3 by having a black bar block the notes beyond that
 - With HR, the playfield "stretches", speeding up notes beyond 16:9, keeping the effective time range to be equal to a maximum of 16:9
 - With HDHR, the HR rule applies and playfield isn't trimmed, but notes fades out much earlier
 
 (Note that these are purely from observations, in code they could be implemented differently)

Perhaps we can name this better - time range has been referred to in aspect ratio equivalents in the community, but it might be better to separate the two concepts here, especially with `trimOnOverflow` set to `true`, the effective time range and display aspect ratio can be decoupled. The issue with this is we are still expressing time range limits in aspect ratio equivalents in the code here, and I'm not sure if there's a better way to express them without introducing more arbitrary-seeming fractions or decimals